app/src/main/java/net/squanchy/support/time/DateTimeFormatter.kt

```diff
@@ -0,0 +1,6 @@
+package net.squanchy.support.time
+
+import org.threeten.bp.format.DateTimeFormatter
+
+fun shortTimeFormatter(): DateTimeFormatter = DateTimeFormatter.ofPattern("h:mm a")
```

Comment:
those are basically backports of what Joda's shortDate and shortTime formatters are.

Reply:
What about `createShortTimeFormatter()`? And, what about using Dagger for this?

Also, this implies the default locale, which is probably what we want for formatting, but we should be careful with something that can also be used to parse. E.g., someone may end up using this `formatter` with their device locale  (say, `it-IT`) to parse a date string from a JSON which will be in `en-US`, and this can cause tricky issues.

Sort of a long shot but I'd have a default parameter `locale: Locale = Locale.getDefault()` to the function to make this a bit more obvious

Reply:
Oh, and please, white line between method implementations

Reply:
872264b